### ISSUE TITLE
Write a Prelimary Email to Send to Companies

### ISSUE THREAD

**Issue by meyerpa** [acm-ndsu/CollabCompetition#11]
Need information about costs for
- Food
- T-Shirts
- Prizes

before writing.

**Comment by matt-fevold** [comment]
I wouldn't bother companies with these details, we just need to ask for money and see what we end up getting.

though we could offer tiers of sponsorship which may come with varying perks (like a logo on a shirt, extra recruitment materials, etc)

**Comment by jghibiki** [comment]
Below are the tiers we considered for the hackathon we had previously attempted to plan. We had companies actually interested in these. Just food for thought.

<div class="panel panel-default">
        <div class="panel-heading"><b>Bronze Tier - $200</b></div> 
        <ul class="list-group">
                <li class="list-group-item">Access to the competition</li>
                <li class="list-group-item">Allowed to distribute recruitment material</li>
                <li class="list-group-item">Provided a copy of the resume's and Github accounts of all participants</li>
        </ul>
</div>

<div class="panel panel-default">
        <div class="panel-heading"><b>Silver Tier - $500</b></div> 
        <ul class="list-group">
                <li class="list-group-item">All the benefits of Bronze Tier</li>
                <li class="list-group-item">Allowed to set up a booth at the competition</li>
        </ul>
</div>

<div class="panel panel-default">
        <div class="panel-heading"><b>Gold Tier - $1000</b></div> 
        <ul class="list-group">
                <li class="list-group-item">All the benefits of Bronze and Silver Tiers</li>
                <li class="list-group-item">Allowed to give a 20 minute presentation at either the opening or closing ceremonies for the event.</li>
                <li class="list-group-item">Allowed to place a company logo in the official event t-shirt.</li>
                <li class="list-group-item">
                        Allowed to place a company logo on the official website for the event.
                        <br>
                        <em>The content of the presentation is up to your company.</em>
                </li>
                <li class="list-group-item">Allowed to distribute prizes outside of the competition format for criteria of your choice.</li>
                <li class="list-group-item">All Gold Tier Sponsors are also allowed to enter a closed envelope bid (supplimentary to the $1000 Gold Tier contribution) to become the title sponsor for the event.</li>
        </ul>
</div>

**Comment by meyerpa** [comment]
I love the tiered system. Regarding the Bronze Tier
- Assumes all participants have GitHub accounts and are willing to provide resumes. This may deter some students from attending the competition. Possibly, could just have access to email addresses and GitHub accounts if available for participants?
- Access to competition may need to be more specific. Are they able to communicate with participants? I would suggest no. Since that's not much different than Silver Tier just without a booth.

Regarding the Gold Tier
- If we have 6 companies with 20 minute presentations, this could add up to two hours of simply sitting there listening to people talk. Maybe we could shorten it to 15 minute presentations?

**Comment by jghibiki** [comment]
> Assumes all participants have GitHub accounts and are willing to provide resumes. This may deter some students from attending the competition. Possibly, could just have access to email addresses and GitHub accounts if available for participants?

Agreed that it is a barrier to entry to some degree, however it is a pretty common requirement for participation in many other competitions. It is a part of the service we can offer the companies is a compilation of resumes. The github account however would have to be optional, as it is not expected that everyone have a github account. On the other hand it is fairly well expected that at some point in college you will need to draft a resume.

> Access to competition may need to be more specific. Are they able to communicate with participants? I would suggest no. Since that's not much different than Silver Tier just without a booth.

The original purpose of this was to have corporate engineers come in and act as roaming mentors for teams, but I agree that the wording is rather ambiguous. 

> If we have 6 companies with 20 minute presentations, this could add up to two hours of simply sitting there listening to people talk. Maybe we could shorten it to 15 minute presentations?

Agreed. I think our original idea was to have a maximum of two gold sponsors, resulting in two 20 min presentations: one at the opening and one at the close. If we want to have more than two, agree that some thought needs to be put into rewording this and/or changing the length of the presentation. Another option would to be to scatter the presentations throughout the competition time and then encourage companies to come up with a novel incentive to draw people away to listen. e.g. energy drinks, doughnut, prizes, etc. This worked really well in a hackathon I went to last year.
